src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/BackupOperation/BackupOperation.cs

```diff
@@ -163,6 +163,16 @@ public string ScriptContent
             }
         }
 
+        public string ErrorMessage
+        {
+            get
+            {
+                return string.Empty;
```

Comment:
I added the error message to IBackupOperation so that the task manager get the error of the operation but didn't integrate the new changes with backup

---
In reply to: [134057582](https://github.com/Microsoft/sqltoolsservice/pull/438#discussion_r134057582) [](ancestors = 134057582)